packages/@aws-cdk/aws-lambda-event-sources/lib/sqs.ts

```diff
@@ -14,6 +14,16 @@ export interface SqsEventSourceProps {
    */
   readonly batchSize?: number;
 
+  /**
+   * The maximum amount of time to gather records before invoking the function.
+   * Maximum of Duration.minutes(5)
+   *
+   * Valid Range: Minimum value of 0 minutes. Maximum value of 5 minutes.
+   *
+   * @default Duration.seconds(0)
```

Comment:
Better to say something like this -

```suggestion
   * @default - no batching window. The lambda function will be invoked immediately with the records that are available.
```